modules/simple_forms_api/app/models/simple_forms_api/vba_40_10007.rb

```diff
@@ -224,6 +224,12 @@ def handle_attachments(file_path)
 
       attachments.each do |attachment|
         combined_pdf << CombinePDF.load(attachment)
+      rescue => e
+        Rails.logger.error(
+          'Simple forms api - failed to load attachment for 40-10007',
+          { message: e.message, attachment: attachment.inspect }
+        )
+        raise
```

Comment:
do we want to propagate the error? or is Rails.logger sufficient?

Reply:
I don't want to lose track of the error itself so I'd rather not change behavior now by swallowing the exception. Maybe we'll discard it in the future if that seems ok but I don't want submissions going through with some but not other attachments going through as well.